lib/archethic/contracts/interpreter/utils.ex

```diff
@@ -270,6 +270,15 @@ defmodule Archethic.Contracts.Interpreter.Utils do
     {node, acc}
   end
 
+  # Whitelist the get_token_id/0 function
```

Comment:
Why using `token_id/0` ?
I think you could use token_id/1 in the Utils, removing the need to integrate it in the condition parser.

Reply:
sorry it's a typo there is no get_token_id/0 only get_token_id/1
but why it would remove the need to integrate it in the condition parser ?

Reply:
I think it could be available anywhere in the actions' parser as well.